app/scripts/controllers/visualizer.js

```diff
@@ -195,6 +195,21 @@ angular.module('openhimConsoleApp')
       }
     };
 
+    // function to add new visualizer to list
+    $scope.addVisualiser = function addVisualiser(){
+
+      // open visualizer settings modal 
+      $modal.open({
+        templateUrl: 'views/visualizerModal.html',
+        controller: 'VisualizerModalCtrl',
+        resolve: {
+          settingsStore: function () { return null; },
```

Comment:
I've added new properties for that, so maybe this one can be removed. Wasn't quite sure how this should have been used when I was doing it.